README.md

```diff
@@ -1,7 +1,10 @@
 # admiral <img src="man/figures/logo.png" align="right" width="200" style="margin-left:50px;"/>
 
 <!-- badges: start -->
-[![Codecov test coverage](https://codecov.io/gh/Roche-GSK/admiral/branch/master/graph/badge.svg)](https://codecov.io/gh/Roche-GSK/admiral?branch=master)
+[<img src="http://pharmaverse.org/shields/admiral.svg">](https://pharmaverse.org)
+[![CRAN status](https://www.r-pkg.org/badges/version/admiral)](https://CRAN.R-project.org/package=admiral)
+[![R-CMD-check](https://github.com/pharmaverse/admiral/workflows/R-CMD-check/badge.svg)](https://github.com/pharmaverse/admiral/actions)
+[![Codecov test coverage](https://codecov.io/gh/Roche-GSK/admiral/branch/main/graph/badge.svg)](https://codecov.io/gh/Roche-GSK/admiral?branch=main)
```

Comment:
Hi @thomas-neitmann ! Should it use `pharmaverse` instead of `Roche-GSK`?

Reply:
It doesn not seem to work when changing to `pharmaverse`. We'll have to investigate further.